app/views/mobile_phone/_form.html.haml

```diff
@@ -0,0 +1,12 @@
+%p
+  My phone number is
+  = telephone_field_tag "mobile_phone[number]", mobile_number, size: 14
```

Comment:
I think we need to use `mobile_phone[full]` here so we get and set the data from the hidden input that is managed by the phone validation widget. This should fix/prevent #209.

Reply:
You can use GitHub's suggestion feature for little tweaks like this :-)  Look:

```suggestion
  = telephone_field_tag "mobile_phone[full]", mobile_number, size: 14
```

Reply:
Hmm, except GitHub is too stupid to allow me to commit that via the web UI, even though I can force-push to this branch :-(